packages/DraftEditor/stories/DraftEditor.stories.tsx

```diff
@@ -10,8 +10,8 @@ const entities = require("entities");
 storiesOf("DraftEditor", module)
   .addParameters({
     readme: {
-      sidebar: DraftEditorReadme
-    }
+      sidebar: DraftEditorReadme,
+    },
```

Comment:
Not sure if you should change something on your lint or prettier setting so that it doesn't add this comma. It's happening across the PR

Reply:
I'll fix it in other pr, we need to improve lint rules